File: app/src/main/java/com/herba/sdk/jetpackexample/roomRecyclerview/Note.java
package com.herba.sdk.jetpackexample.roomRecyclerview;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "notes")
public class Note {


    @PrimaryKey
    @NonNull
    public String id;

    @NonNull
    @ColumnInfo(name = "note")
    public String mNote;

    public Note(String id, String mNote) {
        this.id = id;
        this.mNote = mNote;
    }

    @NonNull
    public String getId() {
        return this.id;
    }

    @NonNull
    public String getmNote() {
        return this.mNote;
    }

}

File: app-kotlin/src/main/java/com/example/myapplication/roomRecyclerview/utils/Note.kt
package com.example.myapplication.roomRecyclerview.utils

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "notes")
class Note(
        @field:PrimaryKey
        var id: String,
        @field:ColumnInfo(name = "note")
        var textNote: String) {

    fun getNote(): String {
        return this.textNote
    }
}